packages/loopback/lib/context/binding.ts

```diff
@@ -5,21 +5,15 @@
 
 export class Binding {
   public value: any;
-  private _isDynamic: boolean = false;
+  public getValue: () => any = () => { throw new Error(`No value was configured for binding {this._key}.`); };
```

Comment:
This is actually giving a useless value because `{this._key}` needs to be `${this._key}`. I'll be fixing this problem in #193